src/operations/blas3/gemm_local.hpp

```diff
@@ -357,7 +393,9 @@ class Gemm<input_t, output_t, DoubleBuffer, NbcA, NbcB, ClSize, TileType,
       auto A = orig_A;
       auto B = orig_B;
       auto C = orig_C;
-      element_t reg_res[item_rows][item_cols] = {};
+      element_t reg_res[item_rows][item_cols];
+      preload_result<check_m_limit, check_n_limit>(reg_res[0], C, ldc, mc, nc,
```

Comment:
you can call it `scaling_c` instead of preload_result

Reply:
Changed!